File: csharp/unittest-practice/Clases/QuesoChihuahua.cs
namespace unittestpractice.Clases
{
    public class QuesoChihuahua : IQueso
    {
        private bool _melted;
        private int _temperature;
        private const int Melting = 20;

        public bool isMelted()
        {
            return _melted;
        }

        public int getCurrentTemperature()
        {
            return _temperature;
        }

        public int getMeltingTemperature()
        {
            return Melting;
        }

        public void setCurrentTemperature(int temp)
        {
            _temperature = temp;
        }

        public void melt(bool melted)
        {
            _melted = melted;

        }
    }
}

File: python/QuesoChihuahua.py
class QuesoChihuahua:

    def __init__(self):
        self.melted = None
        self.meltingTemperature = 20
        self.currentTemperature = None

    def isMelted(self):
        return self.melted

    def getCurrentTemperature(self):
        return self.currentTemperature

    def getMeltingTemperature(self):
        return self.meltingTemperature

    def setCurrentTemperature(self, temp):
        self.currentTemperature = temp

    def melt(self, melted):
        self.melted = melted
